### DISCUSSION TITLE
Should `surround` be visible in which-key?

### DISCUSSION THREAD

**Discussion by florence-wolfe** [LazyVim/LazyVim#214] [Q&A] [answered]
As that title suggests, I'm wondering if `+surround` is supposed to be visible (along with its options) when pressing `<leader>g` in visual mode. I have a recording that shows the behavior. It's almost as if the menu is hidden from which-key but still bound.

![which-key-surround](https://user-images.githubusercontent.com/87673558/217601058-474f2a8c-23e9-4235-94b9-a3af5e31fc66.gif)

Moreover, is it normal behavior for the `<leader>gz` binding to move my visual selection forward by a single character? This makes the surround capture the wrong selection in this case. The same behavior is in the recording as well.

Wondering if these are expected or if should escalate to an issue.

Maybe related to #172 ?

**Comment by folke** [comment, marked as answer]
The bindings are under gz, not leader gz.

Leader is a space, so that's why you go forward one character.

So just don't do leader gz

**Reply by florence-wolfe** [reply]
Yeah that'll do it! 😶‍🌫️ Thank you! :)
